cordova-lib/src/cordova/platform.js

```diff
@@ -663,6 +668,12 @@ function copy_cordovajs_src(projectRoot, platform, platLib) {
     } 
 }
 
+// Remove <platform>.json file from plugins directory.
+function removePlatformPluginsJson(projectRoot, target) {
+    var plugins_json = path.join(projectRoot, 'plugins', target + '.json');
+    if (fs.existsSync(plugins_json)) shell.rm(plugins_json);
```

Comment:
Nit: you can use shell.rm('-f', plugins_json) and then you do not need an exists check.